Rakefile

```diff
@@ -3,7 +3,7 @@ begin
 
   RSpec::Core::RakeTask.new(:spec)
 
-  task :default => :spec
+  task default: :spec
```

Comment:
Sometimes `=>` is used in a DSL sense, in that it expresses an if/then relationship, dependency, or similar.